src/shadowbox/server/manager_service.ts

```diff
@@ -129,6 +129,7 @@ export function bindService(
   );
 
   apiServer.post(`${apiPrefix}/access-keys`, service.createNewAccessKey.bind(service));
+  apiServer.put(`${apiPrefix}/access-keys/:id`, service.createNewAccessKey.bind(service));
```

Comment:
PUT would be to modify a key. To create a new one we should use POST only.

Reply:
POST is to create *a new resource*. PUT is meant for creating (and updating) *a specific resource*. Since we're creating with an identifier here (i.e. the location of the resource), PUT seems more appropriate. That seems more in line with the HTTP spec to me at least.

Reply:
Wow, I think I had the wrong understanding of PUT. TIL.

The [HTTP Semantics RFC](https://www.rfc-editor.org/rfc/rfc9110.html#name-put) was helpful (so was [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PUT)), and it suggested that we may want to use [409 Conflict](https://www.rfc-editor.org/rfc/rfc9110.html#status.409) for pre-existing keys.

Reply:
Done.